src/BcPrebidLoader.js

```diff
@@ -3,171 +3,223 @@
  * Header Bidding Plugin Brightcove module.
  */
 
-// CONSTANTS
-
-var _vjs = window.videojs !== undefined ? window.videojs : null;
+// DEPENDENCIES
 var _prebidGlobal = require('./PrebidGlobal.js');
 var _logger = require('./Logging.js');
-var _prefix = 'PrebidPluginLoader->';
-
-var $$PREBID_GLOBAL$$ = _prebidGlobal.getGlobal();
 
+// CONSTANTS
+var LOADER_VERSION = '0.4.1';
+var PREBID_PLUGIN_ID = 'bcPrebidVastPlugin';
+var COMMAND_PLUGIN_ID = 'bcPrebidVastPluginCommand';
 var DEFAULT_PLUGIN_JS_URL = '//acdn.adnxs.com/video/plugins/bc/prebid/bc_prebid_vast_plugin.min.js';
 
-_logger.always(_prefix, 'Plugin loader version 0.4.1');
+var LOGGER_PREFIX = 'PrebidPluginLoader->';
 
+// OBJECT REFS
+var $$PREBID_GLOBAL$$ = _prebidGlobal.getGlobal();
+var _vjs;
 var _player;
+var _options;
 var _prebidPluginObj;
+var _pluginScrEl;
 
-// register videojs prebid plugins
-function regPrebidVastPlugin(vjs) {
-	_vjs = vjs;
-	// getPlugins not exist in Brightcove Player v5.28.1
-	if (!vjs.getPlugins || !_vjs.getPlugins().bcPrebidVastPlugin) {
-		registerPrebidVastPlugin();
-	}
+// STATE
+window.BCVideo_PrebidPluginApiQue = null;
```

Comment:
Can we use $$PREBID_GLOBAL$$.BCVideo_PrebidPluginApiQue ?

Reply:
Sure, I think that will work. :-)